indexer-app/src/cmr/indexer/data/concepts/collection/collection_util.clj

```diff
@@ -10,3 +10,12 @@
   (try
     (str (Integer/parseInt version-id))
     (catch Exception _ version-id)))
+
+(defn opendap-url?
+  "Determines if the related-url is a OPeNDAP service url."
+  [related-url]
+  (if (and (= (:URLContentType related-url)  "DistributionURL")
```

Comment:
Nit: You can remove the `if`. `and` will return true or false.

Reply:
yea I just realized that, thanks